libraries/opensk/src/env/test/mod.rs

```diff
@@ -19,45 +19,61 @@ use crate::api::connection::{HidConnection, SendOrRecvResult, SendOrRecvStatus};
 use crate::api::crypto::software_crypto::SoftwareCrypto;
 use crate::api::customization::DEFAULT_CUSTOMIZATION;
 use crate::api::firmware_protection::FirmwareProtection;
+use crate::api::rng::Rng;
 use crate::api::user_presence::{UserPresence, UserPresenceResult};
 use crate::api::{attestation_store, key_store};
 use crate::env::Env;
 use customization::TestCustomization;
 use persistent_store::{BufferOptions, BufferStorage, Store};
 use rand::rngs::StdRng;
-use rand::{Rng, SeedableRng};
-use rng256::Rng256;
+use rand::SeedableRng;
+use rand_core::{CryptoRng, Error, RngCore};
 
 pub mod customization;
 mod upgrade_storage;
 
 pub struct TestEnv {
-    rng: TestRng256,
+    rng: TestRng,
     user_presence: TestUserPresence,
     store: Store<BufferStorage>,
     upgrade_storage: Option<BufferUpgradeStorage>,
     customization: TestCustomization,
     clock: TestClock,
 }
 
-pub struct TestRng256 {
+pub struct TestRng {
```

Comment:
I don't remember if we discussed this. But is there a reason not to do `pub type TestRng = StdRng`?

Reply:
Yes, because those additional traits are from the crate itself.